### DISCUSSION TITLE
TFT_Meter_analog repositioning

### DISCUSSION THREAD

**Discussion by g2maurer** [Bodmer/TFT_eSPI#2959] [General]
I would like to be able to reposition the meter to something other than 0,0. I'm using a 320x480 TFT.
I've tried adding an offset for both x and y in all the TFT drawing calls but I can't seem to get the needle
to track properly. The needle overwrites the scale either left or right side depending on what I do.
Here is the example of how I implemented the repositioning offset.
moX for x offset
moY for Y offset

[TFT_Meter_linear.zip](https://github.com/Bodmer/TFT_eSPI/files/13275074/TFT_Meter_linear.zip)

**Comment by Bodmer** [comment]
Use the widget library, this allows multiple meters and to put them in different positions.
https://github.com/Bodmer/TFT_eWidget

**Comment by g2maurer** [comment]
Thanks for your help. I'll look into using Widgets.
